timemachine/fe/free_energy.py

```diff
@@ -163,6 +163,13 @@ def __post_init__(self):
         assert self.ligand_idxs.dtype == np.int32 or self.ligand_idxs.dtype == np.int64
         assert self.protein_idxs.dtype == np.int32 or self.protein_idxs.dtype == np.int64
 
+    def to_bound_impl(self, precision=np.float32):
+        bps = self.potentials
+        summed_pot = SummedPotential([bp.potential for bp in bps], [bp.params for bp in bps])
+        params = np.concatenate([bp.params.reshape(-1) for bp in bps])
+        impl = summed_pot.to_gpu(precision).bind(params).bound_impl
+        return impl
```

Comment:
nit: can be simplified using `make_summed_potential` from `timemachine.potentials`:

```suggestion
        return make_summed_potential(self.potentials).bound_impl
```

Reply:
Ahh, overlooked that -- simplified in https://github.com/proteneer/timemachine/pull/1387/commits/3ef36610d6cb8b51edb3a53eb46769435c8961a2